pkg/appmgmt/general/general.go

```diff
@@ -134,12 +134,18 @@ func (os *Manager) getAppMetadata(pod *v1.Pod) (interfaces.ApplicationMetadata,
 	}
 	ownerReferences := getOwnerReferences(pod)
 
-	placeholderTimeout, err := utils.GetPlaceholderTimeoutParam(pod)
-	if err != nil {
+	placeholderTimeout, placeholderTimeoutErr, schedulingStyle, schedulingStyleErr := utils.GetSchedulingPolicyParam(pod)
```

Comment:
this is a bit confusing. Please create 2 different methods (GetPlaceholderTimeoutParam and GetSchedulingStyle) and eventually export the repeated parts into a separate one.